source/MaterialXGenShader/Factory.h

```diff
@@ -26,52 +26,37 @@ class Factory
 
     /// Register a new class given a unique type name
     /// and a creator function for the class.
-    static void registerClass(const string& typeName, CreatorFunction f)
```

Comment:
No need for members and methods of this class to be static. In fact they way it was used in our code assumes it is _not_ static.